_targets/build.project.armv7a7-imx6ull

```diff
@@ -54,26 +53,28 @@ PREINIT_SCRIPT=(
 	"map ddr 0x80000000 0x87ffffff rwx"
 	"phfs usb0 1.2 phoenixd"
 	"console 0.0"
-	"wait 2000"
 )
 
 
-#TODO: if plo is introduced, BOOT_DEVICE should be changed
 #TODO: if flash driver for imx6ull-evk implementation is introduced, psh should be replaced by imx6ul-flash and psd
 # This is temporary solution for running a system image in RAM
 UPLOAD_SCRIPT=(
-	"kernelimg ${BOOT_DEVICE}"
+	"wait 2000"
```

Comment:
Aren't these scripts used on DTR project? Keep in mind that `_targets` part is common.

Reply:
Since theres no plo on DTR they are only used to create syspage

Reply:
Ok, it seems that syspagen ignores anything beyond `alias`, `map`, `app` and `console` commands, should be ok